lib/flipper/adapters/active_record.rb

```diff
@@ -154,6 +156,10 @@ def get_all
         end
       end
 
+      def rails_version_over_4_2?
+        Rails::VERSION::MAJOR * 10 + Rails::VERSION::MINOR > 42
```

Comment:
```suggestion
        Rails.version.to_f > 4.2
```

If version detection is the only way, this `to_f` trick is a nice way to get the major/minor version.